pages/common/jupyter.md

```diff
@@ -0,0 +1,19 @@
+# jupyter
+
+> Web application for scientific computing and reproducible research.
```

Comment:
This does not mention the primary purpose of jupyter, which is to share documents containing code, equations etc. How about 

```
> Web application to create and share documents that contain code, visualizations and notes.
> Primarily used for scientific computing and machine learning.
``` 
?